src/containers/admin/header/Header.jsx

```diff
@@ -0,0 +1,74 @@
+import React, { useEffect, useState } from 'react';
+import screenfull from 'screenfull';
+import dayjs from 'dayjs';
+import PubSub from 'pubsub-js';
+import { Button } from 'antd';
+import { FullscreenOutlined, FullscreenExitOutlined } from '@ant-design/icons';
+import { reqWeather } from '../../../api/index'
+import './css/header.less';
+export default function Header() {
+  const [isFull, setIsFull] = useState(false)
+  const [date, setDate] = useState(dayjs().format('YYYY年 MM月DD日 HH:mm:ss'))
+  const [weatherInfo, setWeatherInfo] = useState({})
+  const [titleInfo, setTitleInfo] = useState('/admin/home')
+
+  useEffect(() => {
+    setInterval(() => {
+      setDate(dayjs().format('YYYY年 MM月DD日 HH:mm:ss'))
+    }, 1000)
+    getWeather()
+    getTitle()
+  }, [])
+
+
+
+  //Get weather
+  const getWeather = async () => {
+    let result = await reqWeather()
+    setWeatherInfo(result)
+  }
+
+  //Get the title from the URL
+  const getTitle = () => {
+    let pathname = window.location.pathname
+    if (pathname.includes('product')) {
+      setTitleInfo('product')
+    } else {
+      let title = pathname.split('/').reverse()[0]
+      setTitleInfo(title)
+    }
+
+  }
+
+  PubSub.publish('gTitle', getTitle)
+
+  //Full-screen switch
+  const fullScreen = () => {
+    screenfull.toggle();
+    setIsFull(!isFull)
+  };
+
+  return (
+    <header className='header'>
+      <div className='header-top'>
+        <Button size='small' onClick={fullScreen}>
+          {isFull === false ? <FullscreenOutlined /> : <FullscreenExitOutlined />}
+        </Button>
+        <span className='username'>欢迎admin</span>
+        <Button type='link' size='small'>退出登录</Button>
+      </div>
+
+      <div className='header-bottom'>
+        <div className='header-bottom-left'>
+          {titleInfo.toLocaleUpperCase()}
+        </div>
+        <div className='header-bottom-right'>
+          {date}<br />
+          {weatherInfo.city}&nbsp;&nbsp;
+          {weatherInfo.weather} 温度:&nbsp;{weatherInfo.temperature}℃
```

Comment:
Use string template for formatting text with variables, never use `&nbsp;`, it's out-dated